app/views/requests/request/_requestable_list_form.html.erb

```diff
@@ -1,10 +1,10 @@
-<% return unless holdings[mfhd].present? %>
+<% return unless @request.holding.present? %>
```

Comment:
You should remove the local from the calling partial since you are no longer utilizing it.  https://github.com/pulibrary/orangelight/blob/main/app/views/requests/request/_request_form.html.erb#L25. and this line should be changed to also utilize the holding method https://github.com/pulibrary/orangelight/blob/3452a91e2fc1d3d2ac9a50c60fbb26faca601690/app/views/requests/request/_requestable_list_form.html.erb#L2. It's a bit odd to be utilizing both...